game/src/main/scala/io/aigar/model/Team.scala

```diff
@@ -19,15 +19,19 @@ class Teams(tag: Tag) extends Table[Team](tag, "TEAMS") {
 object TeamDAO extends TableQuery(new Teams(_)) {
   lazy val teams = TableQuery[Teams]
 
-  def create(db: Database, team: Team): Team = {
+  /**
+   * This function uses the TableQuery[Teams] to access all the teams and find the one
+   * we just added (with no id) and returns it from the database with the auto-generated id.
+   */
+  def createTeam(db: Database, team: Team): Team = {
```

Comment:
Ok, so if this function fetch the created team with no id, when is that created?